docker/test

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+HERE=$(dirname $0)
+. $HERE/common
+
+SRC=/tmp/src
+DEST=/tmp/dest
+ORDERLY_IMAGE=vimc/orderly:master
+
+mkdir $DEST -p
+mkdir $SRC -p
+
+function cleanup() {
+   rm $SRC -rf
+   rm $DEST -rf
+   docker rm orderly2outpack -f
+}
+
+trap cleanup EXIT
+
+docker pull $ORDERLY_IMAGE
+docker run --rm \
+    --entrypoint 'create_orderly_demo.sh' \
+    -u $UID \
+    -v $SRC:/orderly \
+    -w "/orderly" \
+    $ORDERLY_IMAGE \
+    "."
+
+docker pull $TAG_SHA
+docker run -v $SRC:/orderly:ro \
+           -v $DEST:/outpack \
+           --name orderly2outpack \
+           --env USER_ID=$UID \
+           -d \
+           $TAG_SHA /orderly/demo /outpack --custom="* * * * *"
+
+# this is annoying but can't figure out another way around this
+sleep 60
```

Comment:
This seems like a really long sleep, what is taking the time?

Reply:
The cron job executes every minute, but no way to manually force it to execute immediately...